core/src/main/java/org/apache/carbondata/core/datastore/page/ColumnPage.java

```diff
@@ -187,6 +179,7 @@ public static ColumnPage newPage(TableSpec.ColumnSpec columnSpec, DataType dataT
         case BYTE:
         case SHORT:
         case SHORT_INT:
+        case BOOLEAN:
```

Comment:
Move it before BYTE. Please modify the same in all switch case

Reply:
ok